datasets/us/plum_book/crawler.py

```diff
@@ -20,7 +20,13 @@ def crawl(context: Context):
             agency_name = row.pop("AgencyName")
             position_title = row.pop("PositionTitle")
             appointment_type = row.pop("AppointmentTypeDescription")
+            # 'Expiration date (for term and time-limited appointments)'
+            # The assumption here is that it's the date when the position expires
             expiration_date = row.pop("ExpirationDate")
+            # 'IncumbentBeginDate' is the date when the person started
+            start_date = row.pop("IncumbentBeginDate")
+            # 'IncumbentVacateDate' is the date when the person vacated the position
+            end_date = row.pop("IncumbentVacateDate")
```

Comment:
The way I understand this, it's when the person has to vacate the position - term-limited usually means one term can only be that long. So it applies to the occupancy (aka term) rather than the position.  Either the person then has to be re-appointed, or that person is not allowed to hold that position again for some time or ever. Someone else will be `ADMINISTRATIVE CONFERENCE OF THE UNITED STATES`, `CHAIRMAN` in 2027.

So I think the correctly logic for end date is something like 

    end_date = row.pop("IncumbentVacateDate", row.pop("ExpirationDate"))`

![image](https://github.com/user-attachments/assets/00666411-7663-4ea0-8f87-cfe9bd3cda13)

https://chcoc.gov/sites/default/files/Periodically%20Listing%20Updates%20to%20Management%20%28PLUM%29%20Guidance%20March%202024.pdf